cmd/cstor-volume-mgmt/controller/volume-controller/handler.go

```diff
@@ -484,7 +484,16 @@ func (c *CStorVolumeController) addResizeConditions(
 // configuration on success updates istgt.conf file
 func (c *CStorVolumeController) updateCStorVolumeDRF(
 	cStorVolume *apis.CStorVolume) {
-	err := volume.ExecuteDesiredReplicationFactorCommand(cStorVolume)
+	// make changes to copyCV instead of cStorVolume
+	copyCV := cStorVolume.DeepCopy()
+	var err error
+	if len(copyCV.Spec.ReplicaDetails.KnownReplicas) <
+		len(copyCV.Status.ReplicaDetails.KnownReplicas) {
+		copyCV, err = c.triggerScaleDownProcess(copyCV)
+	} else if len(copyCV.Spec.ReplicaDetails.KnownReplicas) ==
```

Comment:
Why is the condition `==` here and why not `>=` ?

Can you add a comment in he code to describe this branching please.

Reply:
Updated the comments to this function and explained why only `==` and why not `>=` in the code